src/server/seeds/dev/006_00_vessels.js

```diff
@@ -12,17 +12,17 @@ exports.seed = function(knex, Promise) {
       return knex("vessels").insert([
         {
           organization_id: 1,
-          mmsi: 23.33,
+          mmsi: "whatever",
           name: "hyf-vessel1"
         },
         {
           organization_id: 1,
-          mmsi: 45.33,
+          mmsi: "whatever",
           name: "hyf-vessel2"
         },
         {
           organization_id: 1,
-          mmsi: 24.33,
+          mmsi: "whatever",
```

Comment:
The seed values should numeral values. The reason I wrote it should be a string is that the numeral values are not something you want to compute - i.e. it does not make sense to do `phoneNumber + 1`, so phone numbers are usually stored as a string. There might also be situations where phone numbers might contain special characters like `+(45)`. In this case the MMSI is just 9 numbers, but it should still be stored as a string.